Stageplay: add theme

henriqueiamarino opened the pull request:
Stageplay is a blog theme that ideally suits writers and the general public who want their sites formatted as scripts.

[Demo site](https://stageplay.mystagingwebsite.com/)

![screenshot](https://github.com/Automattic/themes/assets/103748085/d7ad679b-7395-4add-a0dd-4efd665460f7)

iamarinoh committed:
Stageplay: add theme

iamarinoh committed:
Stageplay: update readme

jasmussen commented:
**General visual observations**

Looks fantastic, love it. I like the subtle darker background in the gridded header area, the gridlines that invisibly extend from there, the small font, the all-caps, and even the typeface and spacing itself. This is a solid, fantastic theme. 

The demo content on the homepage is _a little long_. I understand it's a full script, but it might be nice to show just a subset, as I'm reviewing I'm not going to read it, I just want to see the footer 😅 

Also, very personal nitpick (feel free to ignore as with all others), I prefer the word "Summary: " over "TLDR:" or "TL;DR:".

Mobile looks fantastic too. I want to use this theme immediately. 

**Template review**

Templates look good. 

The Four Oh Four in ASCII is a bit out there:

![Screenshot 2024-07-10 at 09 46 59](https://github.com/Automattic/themes/assets/1204802/ef663191-6917-497a-b77a-5980d09f8fe3)

I think it's fun. I wonder what theme feedback will say regarding screen readers reading this aloud, you might get pushback. If that happens and you have to remove that ASCII, it will not be the end of the world.

The header navigation font looks different in Playground from the demo site:
![Screenshot 2024-07-10 at 09 48 50](https://github.com/Automattic/themes/assets/1204802/1de7da79-e074-4c08-8ef2-34c6db1d4033)

![Screenshot 2024-07-10 at 09 48 55](https://github.com/Automattic/themes/assets/1204802/c63c0bd5-5485-468f-8775-54598b8a2628)

It looks okay on the frontend, so curious what this is. It may be nothing?

**Style variations**

Style variations look good. I appreciate you included some type changes. 

The default color style, warm white or beige, works great for this. Pink, blue and green, less so. 

![Screenshot 2024-07-10 at 09 51 17](https://github.com/Automattic/themes/assets/1204802/88dd30a1-2b6d-4bd4-8a99-a61aa1aa8c0b)

I had expected some monochrome variations. One that has a pure white background. Perhaps one with a pure black background. perhaps one with a gray background, or even a super dark gray background, maybe even a warm-gray dark background. Quick test:

![Screenshot 2024-07-10 at 09 54 01](https://github.com/Automattic/themes/assets/1204802/f32b776a-445c-4f08-936d-9cc12f2b39d4)

**File and readme reviews**

Readme and licenses look good.

Something is off with the font filenames again:

![Screenshot 2024-07-10 at 09 54 18](https://github.com/Automattic/themes/assets/1204802/e66595c0-0cc2-4d79-9ffe-690e4d2e1c28)

This is also a question for you, for may own education: what are these pattern files? They look like theme template files:

![Screenshot 2024-07-10 at 09 54 24](https://github.com/Automattic/themes/assets/1204802/504bab63-213a-4440-af74-387cc9b7e6c0)

**Summary**

This is a fantastic theme, it will be a hit. There are primarily some technical details to tweak, let me know if I can help there. Then there's the open question of whether to stress the style variations a little bit. Not a blocker, but given how lovely this is, perhaps worth a shot? Nice work!

henriqueiamarino committed:
Stageplay: Fixes following review

henriqueiamarino commented:
- Fixed the font titles; I hope the next CBT update will address that.
- Added a new dark style variation and kept the colorful ones to connect with [these](https://vocal.media/geeks/greatest-film-screenplays-of-all-time) famous NY street movie scripts;
- A light version is already available on a twin theme named [Moviescripts](https://github.com/Automattic/themes/pull/7878);
- I deleted the unused patterns, but I also don't know what they are;
- Fixed the Navigation. The site title and navigation had the same style, but there was a Navigation Link style I got rid of;
- I also removed the ASCII text from the 404 page.

@jasmussen, the preview links are bugged again, as you can see. I didn't touch any of those themes, but they are there. Should I merge this PR or create a new one? I know you mentioned the first, I just want to confirm it here; thank you.

jasmussen commented:
Fantastic followup. Changeset looks great, changes look great. This one is ready! Feel free to press the button, and nice work!
